src/registry.ts

```diff
@@ -56,6 +56,14 @@ export async function getEntry(
         datakey: toHexString(hashDataKey(dataKey)),
       },
       timeout: opts.timeout,
+      // Transform the response to add quotes, since uint64 cannot be accurately
+      // read by JS so the revision needs to be parsed as a string.
+      transformResponse: function (data: string) {
```

Comment:
Hm, well obviously a bit dirty at first sight, but probably there are not many alternative ways of solving this. I'm assuming you spent time googling this right? I do think though that the `.replace` with the mention on `revision` as a field is something we can improve upon.

Why don't we for example use this library: https://www.npmjs.com/package/json-bigint
There are many more like this, might be worth investigating those as well.
This way we could have the SDK handle all future bigints coming from the API in the same way.
By default it uses `bignumber.js` but they have an option `useNativeBigInt` that would work for us I think.

Reply:
I actually checked that library earlier and it's going to pull bignumber.js into the package regardless whether we use it or not so if it's just one use case then I think we should be fine with a replace hack.

Reply:
It seems a bit hacky but this is precisely the intended use of the transform functions. I don't think it's a huge deal.